firebase-firestore/src/test/java/com/google/firebase/firestore/local/SQLiteLocalStoreTest.java

```diff
@@ -14,14 +14,33 @@
 
 package com.google.firebase.firestore.local;
 
+import java.util.Arrays;
+import java.util.Collection;
 import org.junit.runner.RunWith;
-import org.robolectric.RobolectricTestRunner;
+import org.robolectric.ParameterizedRobolectricTestRunner;
 import org.robolectric.annotation.Config;
 
-@RunWith(RobolectricTestRunner.class)
+@RunWith(ParameterizedRobolectricTestRunner.class)
 @Config(manifest = Config.NONE)
 public class SQLiteLocalStoreTest extends LocalStoreTestCase {
 
+  private QueryEngine queryEngine;
+
+  @ParameterizedRobolectricTestRunner.Parameters(name = "QueryEngine = {0}")
```

Comment:
Probably not for this PR but all the different mechanisms we have for parameterizing the local store tests are getting out of hand.

We should have one way of describing the product of persistence x gc x query-engine.